package.json

```diff
@@ -17,6 +17,7 @@
     "node": ">=8"
   },
   "dependencies": {
+    "acorn": "^6.0.4",
```

Comment:
This was missing. Nothing worked without it.

Reply:
That's odd - sounds like it might the same issue as https://github.com/eslint/eslint/issues/11018?

Reply:
That sounds plausible. I will check it again later.

Reply:
The problem is eslint. Running the tests only works... So I moved acorn to dev dependencies until eslint figures out what to do.

Reply:
Noted as TODO in the description, resolved.